python/ray/remote_function.py

```diff
@@ -216,15 +168,23 @@ def f():
             retry_exceptions=retry_exceptions,
             placement_group=placement_group,
             placement_group_bundle_index=placement_group_bundle_index,
-            placement_group_capture_child_tasks=(placement_group_capture_child_tasks),
-            runtime_env=new_runtime_env,
+            placement_group_capture_child_tasks=placement_group_capture_child_tasks,
+            runtime_env=runtime_env,
             name=name,
             scheduling_strategy=scheduling_strategy,
         )
+        # get user input options and override original options
+        _options = options.user_input_filter(_options)
+        _options = {**self._option_dict, **_options}
```

Comment:
nit:  _options and _option_dict sound similar, maybe name then as _options_from_remote and _options_from_options?

Reply:
How is this resolved, seems the code is not changed. (I'm totally fine with rejecting the comment :), just want to see the reason.

Reply:
this is removed